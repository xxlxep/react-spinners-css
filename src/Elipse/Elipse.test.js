import React from 'react';
import renderer from 'react-test-renderer';
import { Elipse } from '..';

it('renders correctly', () => {
  const tree = renderer.create(<Elipse />).toJSON();
  expect(tree).toMatchSnapshot();
});
