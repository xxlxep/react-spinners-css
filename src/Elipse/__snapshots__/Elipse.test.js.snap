// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="lds-css ng-scope"
  >
    <div
      className="lds-ripple"
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div />
      <div />
    </div>
  </div>,
]
`;
